### PR TITLE
Translation Delivery

### PR DESCRIPTION
**Pull request by service-tip-git**
Translation Delivery

### CODE REVIEW

**Comment by service-tip-git** [comment]
This pull request is for translation delivery. Please review and approve them.

File: resources/i18n/en/strings.xml [hunk 1]
```diff
@@ -81,7 +81,7 @@
...
   <string name="VACCINATION_STATE_COMPLETE_IMMUNIZATION_RC_LONG_TEXT">"Based on your vaccination and past infection, you are considered to be fully vaccinated. However, please also find out about the recommended booster vaccinations."</string>
   <!-- XTXT: Vaccination State - PARTIAL_IMMUNIZATION Card Long Text -->
-  <string name="VACCINATION_STATE_PARTIAL_IMMUNIZATION_LONG_TEXT">"You have not received all of the currently recommended vaccinations yet. You do not have full vaccination protection yet."</string>
+  <string name="VACCINATION_STATE_PARTIAL_IMMUNIZATION_LONG_TEXT">"You have not received all of the currently recommended vaccinations yet. You do not have full vaccination protection yet.\n\nPlease note that the vaccination status displayed in the act is determined based on German infection protection law. Children under 12 years of age, as well as all other people, are currently not subject to any restrictions resulting from the “Fully vaccinated” status displayed in the app. Please follow the recommendations of the German vaccination commission, as well as any possibly different rules in your German state."</string>
```
Comment (Ein-Tim) [inline resources/i18n/en/strings.xml:84]: @mlenkeit Do we have a typo here? I think "act" should be "app", right?

**Review by Ein-Tim — COMMENTED** [review]
There is a typo in here.

File: resources/i18n/en/strings.xml (hunk 1 above)
Comment (mlenkeit) [reply resources/i18n/en/strings.xml:84]: @Ein-Tim thanks for bringing this up, I'll created https://github.com/corona-warn-app/cwa-app-ccl/pull/123 to fix it

**Pull request merged**